features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/DefaultRecorderProvider.kt

```diff
@@ -17,14 +17,15 @@ import android.widget.NumberPicker
 import android.widget.RadioButton
 import android.widget.SeekBar
 import android.widget.TextView
+import androidx.appcompat.widget.ActionBarContainer
```

Comment:
interesting enough there is also [android.internal.widget.ActionBarContainer ](https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/com/android/internal/widget/ActionBarContainer.java), should we support it?

Reply:
That one is internal and not visible from customer applications. I'm not sure anyone uses it today, so for now let's say no. Anyway we have some telemetry RFC to list "unsupported" views to know which one we should support.